.github/workflows/update.yml

```diff
@@ -43,14 +44,20 @@ jobs:
         with:
           nix_path: nixpkgs=channel:nixos-22.11
 
-      # - uses: cachix/cachix-action@v12
-      #   with:
-      #     name: acvm-simulator-cache
-      #     authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
+      - name: Build with Nix
+        run: |
+          nix build github:noir-lang/acvm-simulator/jb-nix-flake
```

Comment:
`jb-nix-flake` should branch be taken from environment?

Reply:
Yep this is just for testing purposes